website/src/styles/components/cards.scss

```diff
@@ -27,3 +27,17 @@ section.ds-cards {
     border-radius: 0;
   }
 }
+
+@media screen and (min-width: $md-breakpoint) {
+  section.ds-cards-horizontal {
+    grid-template-columns: 1fr;
+
+    figure {
+      flex-direction: row;
+    }
+  }
+
+  svg {
+    flex: 0 0 30%;
```

Comment:
sth is wrong with ds-cards?
<img width="1074" alt="image" src="https://github.com/sl-design-system/components/assets/111562742/42141297-f7b6-43e1-bd84-39d7d6268136">